README.md

```diff
@@ -60,14 +61,13 @@ Version 4 algorithm is most widely used:
 | v5                | 231        | 1    |
 | v3                | 29         | .1   |
 
-### Reasons for not Supporting Other UUID Versions
+### Regarding other UUID versions
 
-While there is utility in other UUID versions, we are advocating starting with the minimal API
-surface possible, that supports a large percentage of users.
+While there is utility in other UUID versions, we are advocating starting with a minimal API
+surface that supports a large percentage of users _(the string representation of Version 4 UUIDs)._
```

Comment:
Did you mean to replace this parenthesized expression with a sentence?

Reply:
I was meaning to say that:

* we were advocating starting with a minimal API surface that supports a large # of users, i.e.,
* the string representation of Version 4 UUIDs.

Any thoughts about better wording?

Reply:
let's wordsmith more after this is landed, I'm excited to land something we can iterate on.